discv5-utp.md

```diff
@@ -8,6 +8,30 @@ This document specifies an implementation of the [uTP](https://www.bittorrent.or
 
 The Discovery v5 protocol provides a simple and extensible UDP based protocol with robust encryption and resistance to deep packet inspection.  The use of UDP however imposes a tight limit on packet sizes. [Sub-protocols](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#talkreq-request-0x05) which wish to implement functionality that requires transmission of data that exceeds this packet size are forced to implement their own solutions for splitting these payloads across multiple UDP packets.  Packet loss makes this type of solution fragile.  A generic solution that can be reused across different Discovery v5 sub-protocols will improve the overall security and robustness of sub-protocols.
 
+## Basic Goals
+**1.1 Offer a reliable data transfer mechanism for serving data**
```

Comment:
```suggestion
**1.1 Offer a generic protocol for nodes to transfer data payloads that exceed the UDP packet size.**
```

Reply:
Is having reliable data transfer not a motivation for uTP? So if theoretically all the data payloads could fit into the UDP packet size, then we wouldn't need uTP? I was under the impression that we need reliable data transfer since we wouldn't like to lose any content along the way.